Amplitude: Send event_id (#583)
rudderlabs/rudderstack-docs: docs/destinations/analytics/amplitude.mdx

Even if you are not having an enterprise account or groups add-on, RudderStack would add `groups`as a `User Property` on a user's profile with `Group name trait` as its type and `Group value trait` as its value.

Let' say you had defined `Group name trait` as `company_id` and `Group value trait` as `company_name` and made the above call then the user would be associated with a Group name : `RS` and a Group Value : `RudderStack` and you can also see that a user property with type `RS` and value `RudderStack` on user's profile.
  
## Send `event_id` in device mode

> RudderStack supports sending `event_id` to Amplitude in the device mode. It can be added under the `integrations` object and is supported for all the above mentioned API calls, namely, `identify`, `track`, `page`, `screen`, and `group`.

Reviewer: ```suggestion
RudderStack supports sending `event_id` to Amplitude in the device mode. You can include it under the `integrations` object and it is supported for all the above-mentioned API calls, namely, `identify`, `track`, `page`, `screen`, and `group`.
```

> A sample `identify` call with `event_id` looks like the following:

Reviewer: ```suggestion
A sample `identify` call with `event_id` is shown below:
```

```javascript
rudderanalytics.identify(
  "1hKOmRA4el9Zt1WSfVJIVo4GRlm", {
    name: "Alex Keener"
  }, {
    integrations: {
      Amplitude: {

>         event_id: 1234

Reviewer: @ItsSudip is `event_id` a string or an int? if so, should it be presented within quotes, i.e. `"1234"`? cc @ishachopra0401

Reviewer 2: It is an int. No need to provide quotations (").
